rwengine/src/engine/Animator.cpp

```diff
@@ -51,9 +51,6 @@ void Animator::tick(float dt) {
             BoneTransform xform;
             if (b.first->type == AnimationBone::R00) {
                 xform.rotation = kf.rotation;
-            } else if (b.first->type == AnimationBone::RT0) {
-                xform.rotation = kf.rotation;
-                xform.translation = kf.position;
```

Comment:
Why was this removed? This is probably a supported version the fallback below might just be a fallback / non-defined behaviour which is still handled